airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/DebeziumEventUtils.java

```diff
@@ -36,9 +36,10 @@ public static AirbyteMessage toAirbyteMessage(final ChangeEventWithMetadata even
                                                 final CdcMetadataInjector cdcMetadataInjector,
                                                 final ConfiguredAirbyteCatalog configuredAirbyteCatalog,
                                                 final Instant emittedAt,
-                                                final DebeziumPropertiesManager.DebeziumConnectorType debeziumConnectorType) {
+                                                final DebeziumPropertiesManager.DebeziumConnectorType debeziumConnectorType,
+                                                final JsonNode config) {
```

Comment:
Do we want to pass in the entire config here? can we just pass in isEnforceSchema?

That way

Reply:
Ah nvm, I see this is coming from AirbyteDebeziumHandler